src/client/components/inputs/index.tsx

```diff
@@ -222,29 +223,32 @@ export const RawInputField: React.FC<React.InputHTMLAttributes<
 > & {
   label?: string
   errors?: string
+  helperText?: string
   variant?: variantType
   showErrorIcon?: boolean
+  disabled?: boolean
 }> = ({
   errors,
+  disabled,
   showErrorIcon = false,
+  helperText,
   label,
   className,
   variant = 'light',
```

Comment:
do we still need this prop?

Reply:
this was just added. we didn't have it before and we need it according to design. 

I added it separate from ErrorText just to make it more clear.

Reply:
oh.. that makes sense. i'll remove that.